cmd/http/config.json

```diff
@@ -7,7 +7,7 @@
     "LocalReplicaNum": 2,
     "//" : "The number of remote replica from the ones whose latency is more than or equals to RemoteStoreLatencyThresholdInMilliSec",
     "RemoteReplicaNum": 1,
-    "StoreType": "mem",
+    "StoreType": "redis",
```

Comment:
redis as default seems not a wise decision for local dev setting. though redis is right for full-fledged deployment, at 
local dev box it is painful to set up with redis back-ends.
For cloud-deployed env, the whole setting file would be generated anyway and it would come up with redis, so changing in this file has no effect on cloud deployment setting.
Is there a strong justification to switch back to redis, in this local file? If users would use redis as local setup, this should be count as one.

Reply:
I don't have a strong opinion on this. "mem" good for me too.

Reply:
reverted to mem

Reply:
For local testing, I am using redis instead of mem since it might expose some issues than mem does. 

So every time I have to change mem to redis for debugging.